Neos.Flow/Classes/Reflection/DocCommentParser.php

```diff
@@ -115,7 +115,7 @@ public function getDescription()
     protected function parseTag($line)
     {
         $tagAndValue = [];
-        if (preg_match('/@[A-Za-z0-9\\\\]+\\\\([A-Za-z0-9]+)(?:\\((.*)\\))?$/', $line, $tagAndValue) === 0) {
+        if (preg_match('/(@[A-Za-z0-9\\\\]+\\\\[A-Za-z0-9]+)(?:(?:\\(|\s)([^\)]*))?/', $line, $tagAndValue) === 0) {
```

Comment:
This regex captures the test cases in the discussion above, but will blow up if there's a round bracket in the annotation args or comment

Reply:
Phew… those seem far-fetched. Then again, could happen, of course.